hbase-server/src/main/java/org/apache/hadoop/hbase/util/RegionMover.java

```diff
@@ -771,9 +879,17 @@ private ServerName stripServer(List<ServerName> regionServers, String hostname,
   @Override
   protected void addOptions() {
     this.addRequiredOptWithArg("r", "regionserverhost", "region server <hostname>|<hostname:port>");
-    this.addRequiredOptWithArg("o", "operation", "Expected: load/unload/unload_from_rack");
+    this.addRequiredOptWithArg("o", "operation",
+      "Expected: load/unload/unload_from_rack/isolate_regions");
     this.addOptWithArg("m", "maxthreads",
       "Define the maximum number of threads to use to unload and reload the regions");
+    this.addOptWithArg("isolateRegionIds",
+      "Comma separated list of Region IDs hash to isolate on a RegionServer and put "
+        + " region server in draining  mode. This option should only be used with '-o isolate_regions'"
+        + " only. By putting region server in decommission/draining mode, master can't assign any"
```

Comment:
nit: remove extra "only" at the and of the sentence: "This option should only be used with '-o isolate_regions' only"

Reply:
done